packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -39,7 +39,8 @@ Terra.describeViewports('Overlay', ['huge'], () => {
 
       it('Clicks on the Full screen overlay button', () => {
         $('#trigger_fullscreen').click();
-        expect($('#root')).toHaveAttribute('inert', ['', 'true']); // chrome returns true, firefox returns ''
```

Comment:
What error did you get from expect-webdriverio V3? Does it no longer allow to use an array in the second argument for `toHaveAttribute`? I wonder if we should add an example in the [terra-functional-testing V2 upgrade guide](https://engineering.cerner.com/terra-ui/dev_tools/cerner-terra-toolkit-docs/terra-functional-testing/upgrade-guides/version-2-upgrade-guide) so that consumers can follow.

Reply:
> What error did you get from expect-webdriverio V3?

`Expected ['', true] but received 'true'` for chrome.  

> Does it no longer allow to use of an array in the second argument for toHaveAttribute?

It does allow to enter an array as a second argument. But now it doesn't allow to pass an array with possible expectations, rather the whole array should be matching.

Earlier, we used [toHaveAttributeFn](https://github.com/webdriverio/expect-webdriverio/blob/v1.4.1/src/matchers/element/toHaveAttribute.ts#L17). Which used to check if the expectation is present in the array or not.

But now, we use [toHaveAttributeAndValueFn](https://github.com/webdriverio/expect-webdriverio/blob/v1.4.1/src/matchers/element/toHaveAttribute.ts#L17). And this expects the values to be matching. So, when we pass an array as an expectation, then it must be matched with an array.

Reply:
Thanks for the explanation. I can't find any team using `toHaveAttribute` in their tests so they shouldn't run into this so probably don't need to add this in the upgrade guide.